tests/gis_tests/gis_migrations/test_operations.py

```diff
@@ -187,6 +190,25 @@ def test_create_model_spatial_index(self):
         if connection.features.supports_raster:
             self.assertSpatialIndexExists('gis_neighborhood', 'rast', raster=True)
 
+    @skipUnlessDBFeature("supports_3d_storage")
+    @skipIf(spatialite, "Spatialite currently doesn't support altering geometry fields")
```

Comment:
Don't want to add a db feature for that? (Just wondering about you're thinking since I thought we're trying to avoid vendor checks.)